
<!-- badges: start -->

[![Launch CNVpytor Binder](http://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/IdoBar/CNVpytor_binder/main)
<!-- badges: end -->

# Analysis of CNVpytor results (Binder-ready)

This repository was developed for te interactive analysis and visualisation of genome-wide copy number variations (CNV)
from Whole-Genome-Sequencing (WGS) data using [CNVpytor](https://github.com/abyzovlab/CNVpytor) [1].

## Instructions

Launch Binder with this repository by clicking on this badge - [![Launch
CNVpytor Binder](http://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/IdoBar/CNVpytor_binder/main),
then follow the instructions in the [CNVpytor repository](https://github.com/abyzovlab/CNVpytor).  
An example Jupyter notebook is available in the `Example` folder. All the required packages and data files are “bound”
in the repository and you can start working straight away.

For more details and instructions how to setup a similar repository,
please visit [From Zero to Binder in
R\!](https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-r.md)

1. Suvakov M, Panda A, Diesh C, Holmes I, Abyzov A. CNVpytor: a tool for copy number variation detection and analysis from read depth and allele imbalance in whole-genome sequencing. GigaScience. 2021 Nov 1;10(11):giab074. <https://doi.org/10.1093/gigascience/giab074>

